.github/workflows/rust-wasmedge-macro-release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
 
       - name: Dry run cargo publish
         env:
-          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+          CARGO_REGISTRIES_MACRO_TOKEN: ${{ secrets.CARGO_REGISTRIES_MACRO_TOKEN }}
```

Comment:
The `env` name should keep the original naming. And the name of Secrets is changed.

Reply:
I'll fix the issues. Thanks!